services/blockchain-indexer/shared/indexer/transactionProcessor/pos/registerValidator.js

```diff
@@ -30,24 +30,37 @@ const MYSQL_ENDPOINT = config.endpoints.mysql;
 const accountsTableSchema = require('../../../database/schema/accounts');
 const transactionsTableSchema = require('../../../database/schema/transactions');
 const validatorsTableSchema = require('../../../database/schema/validators');
+const commissionsTableSchema = require('../../../database/schema/commissions');
 
 const { getPosConstants } = require('../../../dataService');
 const { requestConnector } = require('../../../utils/request');
 
 const getAccountsTable = () => getTableInstance(accountsTableSchema, MYSQL_ENDPOINT);
 const getTransactionsTable = () => getTableInstance(transactionsTableSchema, MYSQL_ENDPOINT);
 const getValidatorsTable = () => getTableInstance(validatorsTableSchema, MYSQL_ENDPOINT);
+const getCommissionsTable = () => getTableInstance(commissionsTableSchema, MYSQL_ENDPOINT);
 
 // Command specific constants
 const COMMAND_NAME = 'registerValidator';
 
+const getCommissionIndexingInfo = (blockHeader, tx) => {
+	const newCommissionEntry = {
+		address: getLisk32AddressFromPublicKey(tx.senderPublicKey),
+		commission: 10000,
```

Comment:
The default commission value should come from the node. Please do not hard code.

Reply:
registerValidator transaction does not include commission. I have set it to 100% as mentioned in this LIP: https://research.lisk.com/t/introduce-reward-sharing-mechanism/386

`Initially, after registration the commission of a delegate is initialized with 100 percent. Afterwards, it can be decreased by submitting a commission change transaction.`

Reply:
Yes, I'm aware of the same but the default value should be exposed from the node. Could you please request the SDK team to expose the same with an existing/new endpoint?